packages/react-components/theme-designer/src/components/Export/CodeSandbox.tsx

```diff
@@ -0,0 +1,286 @@
+import * as React from 'react';
+import { Link, BrandVariants } from '@fluentui/react-components';
+import { getParameters } from 'codesandbox-import-utils/lib/api/define';
+import * as dedent from 'dedent';
+
+export interface CodeSandboxProps {
+  className?: string;
+  brand: BrandVariants;
+  isLightTheme: boolean;
+}
+
+export const CodeSandbox: React.FC<CodeSandboxProps> = props => {
```

Comment:
Be a little more descriptive in your component names. CodeSandboxLinks, or ExportLinks. ExportLinks is great in that if we ever switched to another service, the name would still work